README.md

```diff
@@ -22,6 +23,15 @@ Run `npm install` to install the required node_modules for this project.
 Run `ng serve` to run the app in dev mode. After executing this command, you can navigate to `http://localhost:4200/` to see the app working.
 The app will automatically reload if you change anything in the source files.
 
+## Prepare your environment for vscode
+
+Install the following extensions:
+
+- `EditorConfig for Visual Studio Code`.
+- `TSLint`
+- `Prettier - Code formatter`.
+- Go to user settings (`settings.json`) and enable formatting on save: `"editor.formatOnSave": true`.
+
```

Comment:
this changes already are in master branch

Reply:
removed.